minetest.conf.example

```diff
@@ -567,6 +567,11 @@
 #      * Only groups which are within the map_generation_limit are generated
 #map_generation_limit = 31000
 
+#    Alternate setting for the generation of non-cubic maps. 
+#      * (Length,Height,Width), or alternatively the sizes of each axis (X,Y,Z).
+#      * map_generation_limit takes precedence, when lower.
+#map_generation_volume = (31000,31000,31000)
```

Comment:
will change to 'extent' for now, but would prefer something that isn't a direct synonym of 'limit'. 'map_generation_limit' setting is already made super redundant and existing just for compatibility, naming the replacement too close seems bad.